.mk/development.mk

```diff
@@ -69,3 +69,35 @@ undeploy-infra: undeploy-loki undeploy-grafana uninstall
 
 .PHONY: undeploy-all
 undeploy-all: undeploy-infra undeploy-sample-cr undeploy-sample-workload
+
+.PHONY: generate-flp-configuration
+generate-flp-configuration: ## Generate custom metrics configuration
+	@echo -e "\n==> Generate metrics configuration"
+	sudo rm -rf /tmp/network_definitions || true
+	sudo rm -rf /tmp/destDir || true
+	cp -r network_definitions /tmp/network_definitions
+	$(OCI_BIN) run --entrypoint /app/confgenerator -v /tmp/network_definitions:/app/network_definitions -v /tmp/destDir:/app/destDir quay.io/netobserv/flowlogs-pipeline:latest --srcFolder /app/network_definitions --destConfFile /app/destDir/flowlogs-pipeline.conf.yaml
```

Comment:
@KalmanMeth can you add also output of documentation, we want to keep it in git and reference that documentation in the readme.md

Reply:
added in config/flp

Reply:
Added Metrics.md